### ISSUE TITLE
No autocomplete when there is a leading parentheses

### ISSUE THREAD

**Issue by waldnercharles** [clinyong/vscode-css-modules#2]
Autocompletion works when doing something like `theme._____` however, when writing `classnames(theme.____)`, there is no popup after the `.`

This can be confirmed by doing `classnames( theme.____)`. Adding a space before the parentheses fixes the issue.

**Comment by waldnercharles** [comment]
This issue can be fixed by changing the `CssModuleCompletionProvider.js`'s `GetWords` function to

```
function getWords(line, position) {
    const text = line.slice(0, position.character);
    const index = text.replace(new RegExp("[a-zA-Z0-9._]*$"), "").length - 1;
    return text.slice(index + 1);
}
```

Instead of testing for only `{` and `(` it now checks for the last instance of any non-alpha numeric character, excluding `.` (dot) and `_` (underscore). The regex can be changed to include or disclude other characters as well.

**Comment by clinyong** [comment]
@waldnercharles Thanks for your suggestion, I will try this later.

**Comment by clinyong** [comment]
This has been solved on [b0f9933b](https://github.com/clinyong/vscode-css-modules/commit/b0f9933b0af621d64520220ea0f7d91ca86b97c7). `Go to definition` also has same problem, and I will republish the extension after solving all these issue.

**Comment by clinyong** [comment]
@waldnercharles All these issue have been solved on version 0.1.2, please update the extension in VSCode.

**Comment by waldnercharles** [comment]
Works great! Thanks.
